### PR TITLE
Add os_version and app_build_version to Sessions

### PR DESCRIPTION
**Pull request by samedson**
#no-changelog

### CODE REVIEW

**Review by mrober — APPROVED** [review]

**Review by visumickey — APPROVED** [review]
These changes look good to me. Let's wait for the sessions event proto changes to land before merging this. Also - I'm guessing there will be a followup PR to use these proto fields to pack the OS version and App Build Version?

**Comment by samedson** [comment]
Yes going to follow up with filling in the fields

**Pull request merged**
